tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_cpu.h

```diff
@@ -344,6 +351,76 @@ class TableWrapperDefault final : public TableWrapperBase<K, V> {
     return Status::OK();
   }
 
+  Status save_to_hdfs(OpKernelContext* ctx, int64 value_dim,
+                      const string& filepath, const size_t buffer_size) {
+    size_t dim = static_cast<size_t>(value_dim);
+    auto lt = table_->lock_table();
+
+    HadoopFileSystem hdfs;
```

Comment:
I think the hadoop file system is moved to `tensorflow/io` after tf 2.6.x. Here might fail with tensorflow 2.6

Reply:
I added a `hadoop_file_system` under the `dynamic_embedding/core/lib` directory so that I don't have to depend on the version of tf or tensorflow/io.